docs/Examples.md

```diff
@@ -24,13 +26,44 @@ eval_dict = {
     'AUC': 1.0
 }
 
+# Tags means 
 tags = {'machine': 'machine01'}
 
+# Model object
 afs_models = models()
+
+# Upload the model to repository and the repository name is the same as file name.
+# Accuracy and loss is necessary, but extra_evaluation and tags are optional.
 afs_models.upload_model(
-    'model.h5', accuracy=0.4, loss=0.3, extra_evaluation=eval_dict, tags=tags)
+    'model.h5', accuracy=0.4, loss=0.3, extra_evaluation=extra_evaluation, tags=tags)
+
+# Get the latest model info 
+model_info = afs_models.get_latest_model_info('model.h5')
+
+# See the model info
+print(model_info)
 ```
 
+**results**
+```
+{
+	'evaluation_result': {
+		'accuracy': 0.4,
+		'loss': 0.3,
+		'confusion_matrix': {
+			'TP': 0.9,
+			'FP': 0.8,
+			'TN': 0.7,
+			'FN': 0.6
+		},
+		'AUC': 1.0
+	},
+	'tags': {
+		'machine': 'machine01'
+	},
+	'created_at': '2018-12-06 08:41:39'
+}```
```

Comment:
The wrong format here. The close of code snippet should be a new line. This causes the following contents also wrong.

Reply:
Thanks.